src/HotChocolate/Core/test/Types.Tests/Types/Descriptors/Conventions/DefaultNamingConventionsTypes.cs

```diff
@@ -5,6 +5,24 @@ namespace HotChocolate.Types.Descriptors
 {
     public class DefaultNamingConventionsTests
     {
+
+        [InlineData("IOFile", "ioFile")]
+        [InlineData("FooBar", "fooBar")]
+        [Theory]
+        public void GetFormattedFieldName_ReturnsFormattedFieldName(string fieldName,string excpectedFieldName) {
```

Comment:
We do have a 100 character max line length
```suggestion
        public void GetFormattedFieldName_ReturnsFormattedFieldName(
              string fieldName,
              string excpectedFieldName) {
```